src/Symfony/Component/Form/FormErrorIterator.php

```diff
@@ -266,6 +267,27 @@ public function seek($position)
     }
 
     /**
+     * Creates iterator for errors with specific codes.
+     *
+     * @param string|string[] $codes The codes to find
+     *
+     * @return static New instance which contains only specific errors.
+     */
+    public function findByCodes($codes)
```

Comment:
As this only works for errors caused by the validator component...im not sure coupling hardcoded is appropriate. It's convenient though..  but maybe this should be `findByCause`..

```php
$errros->findByCause(function($cause) use ($codes) {
    return $cause instanceof ConstraintViolation && in_array($cause->getCode(), (array) $codes, true);
});
```